src/improvements/tasks/sep/005-opcm-upgrade-v300/VALIDATION.md

```diff
@@ -0,0 +1,402 @@
+### [`0x05c993e60179f28bf649a2bb5b00b5f4283bd525`](https://github.com/ethereum-optimism/superchain-registry/blob/00208555c3c356d6596feedb619da989de478ed7/superchain/configs/sepolia/ink.toml#L59-L59) (SystemConfig) - Chain ID: 763373
+
+- **Key:**          `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc`
+  - **Decoded Kind:**      `address`
+  - **Before:** `0x760C48C62A85045A6B69f07F4a9f22868659CbCc`
+  - **After:** `[0x340f923E5c7cbB2171146f64169EC9d5a9FfE647](https://github.com/ethereum-optimism/superchain-registry/blob/00208555c3c356d6596feedb619da989de478ed7/validation/standard/standard-versions-sepolia.toml#L9-L9)`
+  - **Summary:**           ERC-1967 implementation slot
+  - **Detail:**            Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.
+
+  ---
+
+### [`0x05f9613adb30026ffd634f38e5c4dfd30a197fa1`]((https://github.com/ethereum-optimism/superchain-registry/blob/00208555c3c356d6596feedb619da989de478ed7/superchain/configs/sepolia/op.toml#L63-L63)) (DisputeGameFactory) - Chain ID: 11155420
+
+- **Key:**          `0x4d5a9bd2e41301728d41c8e705190becb4e74abe869f75bdb405b63716a35f9e`
+  - **Before:**     `0x0000000000000000000000007717296cac5d39d362eb77a94c95483bebae214e`
+  - **After:**     `0x000000000000000000000000845e5382d60ec16e538051e1876a985c5339cc62`
+  - **Summary:** Set a new game implementation for game type [PERMISSIONED_CANNON.](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v2.0.0-rc.1/packages/contracts-bedrock/src/dispute/lib/Types.sol#L55")
+    - **Detail:** You can verify this slot corresponds to the game implementation for game type 1 by 
+                      deriving the slot value as follows:
+                      - Notice that [`gameImpls`](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v2.0.0-rc.1/packages/contracts-bedrock/src/dispute/DisputeGameFactory.sol#L57") is a map from a `GameType` to a dispute game address.
+                      - Notice that `GameType` is [equivalent to a]("https://github.com/ethereum-optimism/optimism/blob/op-contracts/v2.0.0-rc.1/packages/contracts-bedrock/src/dispute/lib/LibUDT.sol#L224-L224").
+                      - Notice that the `gameImpls` [stored at slot 101]("https://github.com/ethereum-optimism/optimism/blob/op-contracts/v2.0.0-rc.1/packages/contracts-bedrock/snapshots/storageLayout/DisputeGameFactory.json#L41-L41"). 
+                      - Calculate the expected slot for game type 1 using `cast index &lt;KEY_TYPE&gt; &lt;KEY&gt; &lt;SLOT_NUMBER&gt;`:
+                        - `cast index uint32 1 101`
+                      - You should derive a value matching the "Raw Slot" here: 0x4d5a9bd2e41301728d41c8e705190becb4e74abe869f75bdb405b63716a35f   
+
+  - **Summary:**  Set a new game implementation for game type [CANNON]("https://github.com/ethereum-optimism/optimism/blob/op-contracts/v2.0.0-rc.1/packages/contracts-bedrock/src/dispute/lib/Types.sol#L52")
```

Comment:
Wheres the state diff for this? Shouldn't it include `0xffdfc1249c027f9191656349feb0761381bb32c9f557e01f419fd08754bf5a1b`?

Reply:
I removed this section.